### PR TITLE
fix(extends): reordered configs at extends block for apply correct ov…

### PR DESCRIPTION
**Pull request by votintsev**
…erride. Removed duplicate rule. *

Duplicate with standard

### CODE REVIEW

File: base.js [hunk 1]
```diff
@@ -33,8 +33,6 @@ module.exports = {
     'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     // Enforce import order
     'import/order': 'error',
-    // Imports should come first
-    'import/first': 'error',
```
Comment (nysos3) [inline base.js]: that's import/order vs import/first. 2 different rules.

**Review by nysos3 — CHANGES_REQUESTED** [review]
Add the import/first rule back.

File: base.js (hunk 1 above)
Comment (votintsev) [reply base.js]: It is duplicate with standard:

https://github.com/standard/eslint-config-standard/blob/f7d50e01310866458fd844625cf588d4ee39823b/eslintrc.json#L190

Comment (nysos3) [reply base.js]: Still prefer being explicit.

**Review by nysos3 — APPROVED** [review]

**Pull request merged**